artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis

```diff
@@ -105,6 +105,16 @@ if [ -z "$LOG_MANAGER" ] ; then
   LOG_MANAGER="$ARTEMIS_HOME/lib/${logmanager}"
 fi
 
+# determine which native library version to load
+LIBRARY_PATH="-Djava.library.path=\"$ARTEMIS_HOME/bin/lib"
+if [ "$(uname -m)" = "x86_64" ]; then
+  LIBRARY_PATH="$LIBRARY_PATH/linux-x86_64\""
+elif [ "$(uname -m)" = "i686" ]; then
+  LIBRARY_PATH="$LIBRARY_PATH/linux-i686\""
+else
+  LIBRARY_PATH=""
```

Comment:
Actually no, the ./artemis create will use the libaio to determine the journal type (if it can't load libaio, it won't use it).. and it will also perform a sync on the system to calculate the timed-buffer values.